components/centraldashboard/public/components/main-page.js

```diff
@@ -254,6 +259,39 @@ export class MainPage extends mixinBehaviors([AppLocalizeBehavior], utilitiesMix
         this.securityMessages = securityMessages || [];
     }
 
+    /**
+     * @param {Event} ev AJAX-response
+     */
+    _onHasDashboardBannerError(ev) {
+        const error = ((ev.detail.request||{}).response||{}).error ||
+            ev.detail.error;
+        this.showError(error);
+        return;
+    }
+
+    /**
+     * Set state for Central dashboard Banner
+     * @param {Event} ev AJAX-response
+     */
+    _onHasDashboardBannerResponse(ev) {
+        const rawBannerMessages = ev.detail.response;
+        const bannerMessages = [];
+        // Filter out the out of date messages
+        rawBannerMessages.forEach((msg) => {
+            const start = new Date(msg['start-date']);
+            const end = new Date(msg['end-date']);
+            const now = new Date(Date.now());
+
+            if ((end > now) && (now >= start)) {
```

Comment:
Should it be `(end >= now)` ? So with the equals sign?

I dont know if this only accepts datetimes, or if it can also accept just dates. Im just thinking of if we ever have a message to add to a banner for just 1 day, where it would be an equal value if there is no time to the dates.